### PR TITLE
feat: :zap: add a ticketswap stripe-tap

### PR DESCRIPTION
**Pull request by tobiascadee**
add a stripe tap that uses the latest Meltano SDK version in combination with the stripe python library. this creates a simple to use and understand tap.

### CODE REVIEW

File: _data/meltano/extractors/tap-stripe/ticketswap.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+capabilities:
+- about
+- batch
+- catalog
+- discover
+- schema-flattening
+- state
+- stream-maps
+description: A Meltano SDK tap for Stripe data.
```
Comment (edgarrmondragon) [inline _data/meltano/extractors/tap-stripe/ticketswap.yml:9]:
```suggestion
description: Online payment processing for internet businesses
```

File: _data/meltano/extractors/tap-stripe/ticketswap.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+domain_url: ticketswap.com
```
Comment (edgarrmondragon) [inline _data/meltano/extractors/tap-stripe/ticketswap.yml:10]:
```suggestion
domain_url: https://stripe.com/docs/api
```

Comment (tobiascadee) [reply _data/meltano/extractors/tap-stripe/ticketswap.yml:10]: oops, thanks!

**Review by edgarrmondragon — APPROVED** [review]
Thanks @tobiascadee!

**Pull request merged**